bot/magik.go

```diff
@@ -16,8 +16,7 @@ func (args MagikArgs) GetImageURL() string {
 	return args.ImageURL
 }
 
-// Magik runs content-aware scaling on an image.
-func Magik(wand *imagick.MagickWand, args MagikArgs) ([]*imagick.MagickWand, error) {
+func liquidRescaleHelper(wand *imagick.MagickWand, args MagikArgs, wMultiplier float64, hMultiplier float64) ([]*imagick.MagickWand, error) {
```

Comment:
Can this function be renamed to better clarify that it's doing something other than liquid rescaling? I'd be fine with something like `magikHelper`